git_machete/cmd.py

```diff
@@ -3478,7 +3544,7 @@ def strip_first_fragment(rb):
             params = parse_options(args, "d:Mn", ["down-fork-point=", "merge", "no-edit-merge", "no-interactive-rebase"])
             read_definition_file()
             expect_no_operation_in_progress()
-            slide_out(params or [current_branch()])
+            slide_out(params or [current_branch()])  # type: ignore
```

Comment:
Why ignore? `tuple` vs `list` again?

Reply:
wrong type of params and [current_branch()], fixed